src/bp-forums/classes/class-bbpress.php

```diff
@@ -483,7 +483,7 @@ public static function register_post_types() {
 						'capabilities'        => bbp_get_topic_caps(),
 						'capability_type'     => array( 'topic', 'topics' ),
 						'menu_position'       => 555555,
-						'has_archive'         => ( 'forums' === bbp_show_on_root() ) ? bbp_get_topic_archive_slug() : false,
+						'has_archive'         => true,
```

Comment:
```suggestion
						'has_archive'         => ( 'forums' === bbp_show_on_root() ) ? true : false,,
```

Can't we do this? Need to check why bbpress have this condition is there any option like not to be archive page

Reply:
@chetansatasiya Okay looking into it.

Reply:
@chetansatasiya I checked with bbpress, the option to choose what should be on the forum root.
- Forum or
- Topics
![image](https://user-images.githubusercontent.com/16851407/126952203-64b620e0-4fd7-48ee-8d17-2ba6a917f70e.png)
If we select the first option "Forum Index", so It means if the Forum root is showing forums, so there should be a separate archive for topics, whereas if we select the second option "Topics by Last Post", so it will show topics on forums page so there is no need for a separate topics archive. That's what the below line of code says 
`'has_archive'         => ( 'forums' === bbp_show_on_root() ) ? bbp_get_topic_archive_slug() : false,`
and in bbpress the pagination shows the the pagination link something like this.